WordPressKit/WordPressKitTests/MockServiceRequest.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  MockServiceRequest.swift
+//  WordPressKitTests
+//
+//  Created by Daniele Bogo on 17/04/2018.
+//  Copyright © 2018 Automattic Inc. All rights reserved.
+//
+
```

Comment:
This one slipped (we're not really using Headers) 😄

Reply:
Argh! I missed this! Thanks! :D